helix-term/tests/test/commands.rs

```diff
@@ -640,3 +640,27 @@ async fn test_join_selections_space() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn test_read_file() -> anyhow::Result<()> {
+    let mut file = tempfile::NamedTempFile::new()?;
+    let contents_to_read = String::from("some contents");
+    let output_file = helpers::temp_file_with_contents(&contents_to_read)?;
+
+    test_key_sequence(
+        &mut helpers::AppBuilder::new()
+            .with_file(file.path(), None)
+            .build()?,
+        Some(&format!(":r {:?}<ret><esc>:w<ret>", output_file.path())),
+        Some(&|app| {
+            assert!(!app.editor.is_err(), "error: {:?}", app.editor.get_status());
+        }),
+        false,
+    )
+    .await?;
+
+    let expected_contents = format!("{}\n", contents_to_read);
+    helpers::assert_file_has_content(&mut file, &expected_contents)?;
```

Comment:
This is failing on the windows CI since windows uses CRLF line endings. You can use the `LineFeedHandling` enum and `apply` to add the line ending here instead of `format!`

Reply:
pushed